src/Tribe/API.php

```diff
@@ -343,8 +343,10 @@ private static function saveEventVenue( $data, $post = null, $post_status = 'pub
 		public static function createOrganizer( $data, $post_status = 'publish' ) {
 			if ( ( isset( $data['Organizer'] ) && $data['Organizer'] ) || self::someOrganizerDataSet( $data ) ) {
 
+				$organizer_label = tribe_get_organizer_label_singular();
+
 				$postdata = array(
-					'post_title'  => $data['Organizer'] ? $data['Organizer'] : "Unnamed Organizer",
+					'post_title'  => $data['Organizer'] ? $data['Organizer'] : sprintf( __( 'Unnamed %s', 'tribe-events-calendar' ), ucfirst( $organizer_label ) ),
```

Comment:
we changed the text domain since you made this pull request.  it is now `the-events-calendar`